internal/encoded.go

```diff
@@ -27,6 +27,8 @@ import (
 	"go.uber.org/cadence/internal/common/util"
 )
 
+//go:generate mockery --name Value --output ../mocks --boilerplate-file ../LICENSE
```

Comment:
I moved these so they're in the file with the thing they define (so the dependency setup is trivial), and because it's simpler: no need to specify the internal-package bit.

code generation after this is identical.